### ISSUE TITLE
After update to firebase-tools: 7.16.0 emulated callable function context does not includes uid

### ISSUE THREAD

**Issue by kamil-ignac-ih21** [firebase/firebase-tools#2057]
### Environment info

**firebase-tools:** 7.16.0

**Platform:** macOS

### Test case

1. setup simple WEB project
2. add callable function
3. run callable function in local emulator
4. sign in via phone number on client side
5. call callable function
6. `functions.https.CallableContext` contains `uid` and `token.uid`

### Steps to reproduce

Have firebase-tools 7.16.0
1. setup simple WEB project
2. add callable function
3. run callable function in local emulator
4. sign in via phone number on client side
5. call callable function
-> `functions.https.CallableContext` doesn't contain `uid`. `token` object contains `user_id` but not `uid`.

### Expected behavior

Function context (+ auth token) includes `uid` of signed user.

### Actual behavior

Current `context.auth` looks like:
```
{
    "token": {
        "iss": "https://securetoken.google.com/d21p-dev",
        "aud": "d21p-dev",
        "auth_time": 1585044490,
        "user_id": "ASyoY08brfRoWNxRaBQJVLK4G012",
        "sub": "ASyoY08brfRoWNxRaBQJVLK4G012",
        "iat": 1585044490,
        "exp": 1585048090,
        "phone_number": "+420777123456",
        "firebase": {
            "identities": {
                "phone": [
                    "+420777123456"
                ]
            },
            "sign_in_provider": "phone"
        }
    }
}
```

**Comment by pkfms** [comment]
Same issue here. auth is undefined in context object

**Comment by samtstern** [comment]
@kamil-ignac-ih21 @pkfms did this work for you in a previous version of `firebase-tools`?  Also could you run the emulator with `--debug` and show me logs from one function call?

**Comment by pkfms** [comment]
I can access context.auth with firebase-tools 7.15.1
attached log from functions emulator. with console.log of context
function crash when trying to access context.auth.uid
[functions-emulator-log.txt](https://github.com/firebase/firebase-tools/files/4375339/functions-emulator-log.txt)

**Comment by kamil-ignac-ih21** [comment]
@samtstern yesterday I was on 7.15.1 and it worked. During the day I moved to 7.16 (+ of course I did several other changes) and it stoped working. But now when I move back to 7.15.1 it still not working.

Here is firebase-debug.log file.
[firebase-debug.log](https://github.com/firebase/firebase-tools/files/4375375/firebase-debug.log)

**Comment by kamil-ignac-ih21** [comment]
@samtstern and here is emulator with `--debug` output.
[emulator-debug.log](https://github.com/firebase/firebase-tools/files/4375385/emulator-debug.log)

**Comment by samtstern** [comment]
@kamil-ignac-ih21 this should be fixed be #2058

**Comment by kamil-ignac-ih21** [comment]
@samtstern thanks for quick update. Any ETA to be released?

**Comment by samtstern** [comment]
Sorry no ETA but we try to release fairly frequently, so I don't think it will be long.

**Comment by samtstern** [comment]
FYI: as a quick fix you can use `auth.token.sub` in place of `auth.uid`.  In production they are the same.

**Comment by kamil-ignac-ih21** [comment]
@samtstern perfect, thank you!

**Comment by samtstern** [comment]
@kamil-ignac-ih21 this should be fixed ini `7.16.1` which was just released.

**Comment by kamil-ignac-ih21** [comment]
@samtstern perfect! Many thanks for so quick fix! Appreciate that!
